src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -1,5 +1,5 @@
 using Random:
-    AbstractRNG, MersenneTwister, randperm, seed!, shuffle!
+    GLOBAL_RNG, AbstractRNG, MersenneTwister, randperm, seed!, shuffle!
```

Comment:
Hm. I'm not sure this is the correct way to do it (could be wrong). Take a look at 
https://github.com/JuliaGraphs/LightGraphs.jl/blob/master/src/SimpleGraphs/generators/randgraphs.jl#L42 *et seq.*. Specifically, we have a `rng = getRNG(seed)` call there that will return the proper RNG.

Reply:
The way they've done it doesn't assume you want Mersenne Twister, though. It might be more general...

Reply:
That's a great point, but we ought to be consistent even if it's not yet ideal. I think we should open a new PR to change the way we handle RNGs in general, and we can have a discussion with folks who are interested in randomness who might not be tied in to this particular PR.

Reply:
We are now using getRNG() with a seed passed in, to be consistent with RNG handling, and our changes have been pushed.